### ISSUE TITLE
Kotlin Android Extensions plugin is missing.

### ISSUE THREAD

**Issue by michaldrabik** [viniciussoares/ribot-android-boilerplate-kotlin#1]
Kotlin extensions plugin is missing, so Android Studio reports them as missing reference.

Needed to add classpath dependency in project's `build.gradle`: 

```
buildscript {
  ext.kotlin_version = '1.0.4'
  repositories {
    jcenter()
  }
  dependencies {
    classpath 'com.android.tools.build:gradle:1.1.3'
    classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
    classpath "org.jetbrains.kotlin:kotlin-android-extensions:$kotlin_version"
  }
}
```

and apply in app's `build.gradle`:

```
apply plugin: 'com.android.application'
apply plugin: 'kotlin-android'
apply plugin: 'kotlin-android-extensions'

android {...}
```

And it's all good now.

**Comment by viniciussoares** [comment]
Hello Michal, how are you? Hope you're doing well :)

The project as it is gives you a warning or you've added something else?

I don't think we're using any of the kotlin-android-extension features. If you create a pull request that showcase any of its features, i'll be happy to merge it :)

**Comment by michaldrabik** [comment]
I'm not sure what even happened. It might be Android Studio 2.2.2 trolling me. I was not able to use Kotlin's language extension features because AS reported that it can't find reference (for example to `Context.getApplicationComponent()`). Then I added extensions plugin to add some of my own stuff and it clicked so I assumed it was the cause. Now I removed extensions plugin and its still working fine.

To sum up this whole issue should've never been there. Cheers

**Comment by viniciussoares** [comment]
I'm glad that your problem is solved.

Feel free to reopen this issue if it happen again :)
